cleverhans/utils.py

```diff
@@ -253,3 +165,17 @@ def grid_visual(data):
     # Draw the plot and return
     plt.show()
     return figure
+
+
+def conv_2d(*args, **kwargs):
+    from cleverhans.utils_keras import conv_2d
+    warnings.warn("utils.conv_2d is deprecated and may be removed on or after"
```

Comment:
It is more of a structure question for what should be included in the `cleverhans/` folder, since we will have to keep keras as a dependency for our Travis setup because of the tests related to the tutorials.

I don't have a strong opinion either way.